libs/openFrameworks/types/ofRectangle.h

```diff
@@ -754,6 +754,16 @@ class ofRectangle{
     /// \returns The height of the rectangle.
     float getHeight() const;
 
+    
+    /// \brief Maps a normalized coordinate into this rectangle
+    /// \returns The mapped coordinate
```

Comment:
Also, in addition to the brief, could you paste a short paragraph (like the above documentation) that expands upon the brief just a bit.